2020/day1/solution.py

```diff
@@ -1,15 +1,14 @@
 file = open('input.txt')
-numbers = map(int, file.readlines())
-number_map = dict(map(lambda x : (x, 1), numbers))
-for num in number_map:
-    if 2020 - num in number_map:
+numbers = set(map(int, file.readlines()))
+for num in numbers:
```

Comment:
This isn't totally correct - imagine the input:

```
1010
3
```

This will "find" 1010 and 1010 even though 1010 only occurs once

Reply:
good point